server/api/utils/email.ts

```diff
@@ -0,0 +1,85 @@
+import nodemailer from "nodemailer";
+
+interface Item {
+    name: string;
+    condition: string;
+    age: number;
+    quantity: number;
+    description: string;
+}
+
+const getItemAgeDescription = (age: number): string => {
+    const descriptionArr = ["Less than a year", "1 year", "2 years", "3 years", "More than 3 years"];
+    if (age < 0) {
+        return descriptionArr[0];
+    }
+    if (age >= descriptionArr.length) {
+        return descriptionArr[descriptionArr.length - 1];
+    }
+    return descriptionArr[age];
+};
+
+const transporter = nodemailer.createTransport({
+    service: "gmail",
+    auth: {
+        user: "blueprintpregnancycentre@gmail.com", // TODO: Change later once TPC gives us an email address
+        pass: process.env.EMAIL_PASSWORD
+    }
+});
+
+async function sendApprovalEmail(firstName: string, lastName: string, email: string, item: Item) {
+    let htmlString = `<body><p>Dear ${firstName} ${lastName}, <p>`;
```

Comment:
I think you can do multiline strings like this:
```
let htmlString = `<body><p>Dear ${firstName} ${lastName}, <p>
<p>After reviewing your donation form, we would like to accept the following items to meet a client's needs.</p>
`
```

Reply:
note sure tho

Reply:
Updated.